### PR TITLE
Add near-matching decomped functions for SeqPlayer

### PR DESCRIPTION
**Pull request by MNGoldenEagle**

### CODE REVIEW

**Comment by MNGoldenEagle** [comment]
Kicking off the review process on these.  Functionally matching, but not OK.  Probably needs some cleanup before merge-worthy.

**Comment by ethteck** [comment]
You have unused asm files (or references to them) that you'll need to delete for Jenkins to pass on this

File: src/code/audio_seqplayer.c [hunk 1]
```diff
@@ -11,6 +11,25 @@
 
 extern u8 D_80130470[];
 
+// static u8 D_80130470[] = {
+// /* 0x00 */ 0x89, 0xC0, 0x85, 0x84, 0x82, 0x77, 0x80, 0x9F, 0x80, 0x01, 0x80, 0x9F, 0x82, 0x77, 0x85, 0x84,
+// /* 0x10 */ 0x89, 0xC0, 0x8F, 0x1F, 0x95, 0x94, 0x9D, 0x0F, 0xA5, 0x7F, 0xAE, 0xCD, 0xB8, 0xE4, 0xC3, 0xAA,
+// /* 0x20 */ 0xCF, 0x05, 0xDA, 0xD9, 0xE7, 0x08, 0xF3, 0x75, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```
Comment (fig02) [inline src/code/audio_seqplayer.c:17]: if this is already migrated to C, id prob just use it and keep it non static. assuming thats the only reason its commented out rn

Comment (fig02) [reply src/code/audio_seqplayer.c:17]: looking at it further im pretty confused as to whats going on, is this data part of another file thats not split yet?

File: src/code/audio_seqplayer.c [hunk 2]
```diff
@@ -956,11 +976,797 @@ void Audio_SequenceChannelSetVolume(SequenceChannel* seqChannel, u8 volume) {
...
+                        channel->pad_22 = (parameters[0] == 0) ? gAudioContext.gAudioRandom // odd load here
+                                                                : gAudioContext.gAudioRandom % parameters[0];
+                        break;
+                    case 0xB8:
+                        scriptState->value = (parameters[0] == 0) ? gAudioContext.gAudioRandom
+                                                                : gAudioContext.gAudioRandom % parameters[0];
+                        break;
+                    case 0xBD: {
+                        result = Audio_NextRandom();
+                        channel->pad_22 = (parameters[0] == 0) ? (u32) result : (u32)result % parameters[0];
```
Comment (isghj5) [inline src/code/audio_seqplayer.c:1358]: I haven't found a vanilla seq that uses a B1-F command yet, so I don't know what this really does, but this is clearly not a pad variable, right? Should it at least get renamed to unk22?

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**